client/src/components/Shared/CircleImg/CircleImg.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Wrapper, Title, Subtitle, Image, PlusCircle } from './CircleImg.styled';
+
+const CircleImg = ({ type, size, imgUrl, title, subtitle, clickHandler }) => (
+	<Wrapper>
+		{title && <Title>{title}</Title>}
+		{type !== 'wPlus' ? (
+			<Image type={type} size={size} onClick={clickHandler} />
+		) : (
+			<PlusCircle size={size}>+</PlusCircle>
+		)}
+
+		{subtitle && <Subtitle>{subtitle}</Subtitle>}
+	</Wrapper>
+);
+
+CircleImg.propTypes = {
+	type: PropTypes.string.isRequired, // round | w plus
```

Comment:
Also here, this could use the oneOf prop type.

Reply:
This is too WIP, the component is already CircleImg so giving it a `round` type is a bit unnecessary, `wPlus` is valid type on the other hand, I'll add it on `oneOf`.